dev/AppNotifications/AppNotificationBuilder/AppNotificationBuilder.cpp

```diff
@@ -31,7 +31,7 @@ namespace winrt::Microsoft::Windows::AppNotifications::Builder::implementation
     {
         THROW_HR_IF_MSG(E_INVALIDARG, key.empty(), "You must provide a key when adding an argument");
 
-        m_arguments.Insert(key, value);
+        m_arguments.Insert(EncodeArgument(key.c_str()), EncodeArgument(value.c_str()));
```

Comment:
Here too. If you can pass this by reference here, you can avoid creating unnecessary copies

Reply:
Changing EncodeArgument param from std::wstring -> winrt::hstring to avoid this